synapse/handlers/room_member.py

```diff
@@ -1237,6 +1242,13 @@ async def _make_and_store_3pid_invite(
         if room_name_event:
             room_name = room_name_event.content.get("name", "")
 
+        room_type = None
+        room_create_event = room_state.get((EventTypes.Create, ""))
+        if room_create_event:
+            room_type = room_create_event.content.get(
+                EventContentFields.ROOM_TYPE, None
```

Comment:
for a `dict`, the second arg to `.get` defaults to `None` already

```suggestion
                EventContentFields.ROOM_TYPE
```

Reply:
Shows how much (little) I do Python :P
I wonder if this is going to trip the linter into wanting it back on L1248

Reply:
I find this to be more of a style thing, some people like to be explicit that it will return `None` in the default case. I was actually going to comment and then deleted it. 😄